commands/src/api/mod.rs

```diff
@@ -12,16 +12,35 @@ pub enum Error {
     DeserializeCommand,
 }
 
-/// API request packet type
+/// API message type
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(untagged)]
+pub enum Message {
+    /// Request
+    Request(Request),
+    /// Notification
+    Notification(Notification),
+}
+
+/// Request type
+/// Expect response from remote end
 #[derive(Debug, Serialize, Deserialize)]
 pub struct Request {
     /// Request body
     pub body: Body,
     /// Response correlation string
-    pub respond_to: Option<String>,
+    pub respond_to: String,
+}
+
+/// Notification
+/// Fire and forget packet
+#[derive(Debug, Serialize, Deserialize)]
+pub struct Notification {
+    /// Request body
+    pub body: Body,
 }
 
-/// API request body
+/// API message body payloads
 #[derive(Debug, Serialize, Deserialize)]
 pub enum Body {
```

Comment:
Thinking that we should split bodies to `RequestBody` and `NotificationBody` as well. Motivation is to define more strictly which types can be sent as requests and which as notifications. What do you think?

Reply:
Well, we can postpone this because `Notification`s are all likely to be replaced with requests in nearest future.